docs/cedarling/reference/cedarling-interfaces.md

```
@@ -9,7 +9,7 @@ tags:
 
 # Cedarling Interfaces
 
-Cedarling provides a number of methods to interface with the Cedar engine. 
+Cedarling provides a number of methods to interface with the Cedar engine.
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 4

(MD009, no-trailing-spaces)